doc/source/user_guide/dsintro.rst

```diff
@@ -397,6 +397,30 @@ The result will be a DataFrame with the same index as the input Series, and
 with one column whose name is the original name of the Series (only if no other
 column name provided).
 
+
+.. _basics.dataframe.from_list_namedtuples:
+
+From a list of namedtuples
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first namedtuple will be used to determine the columns of the DataFrame.
```

Comment:
I might write:
```
The field names of the first ``namedtuple`` in the list determine the columns of the ``DataFrame``. 
The remaining namedtuples (or tuples) are simply unpacked and their values are fed
into the rows of the ``DataFrame``. If any of those tuples is shorter than the first ``namedtuple`` 
then the later columns in the corresponding row are marked as missing values or, in case 
it is longer than the first ``namedtuple``, a ``ValueError`` is raised.
```